tests/config/functional/toml/issue_3181/toml_decode_error.1.out

```diff
@@ -1,2 +1,2 @@
 ************* Module {abspath}
-{relpath}:1:0: F0011: error while parsing the configuration: Found invalid character in key name: '*'. Try quoting the key name. (line 3 column 2 char 48) (config-parse-error)
+{relpath}:1:0: F0011: error while parsing the configuration: Invalid statement (at line 3, column 1) (config-parse-error)
```

Comment:
Well, it's a little sad to see the default builtin lib have worst error messages by far than the old one.